usmqe/web/skyring/mainpage/navpage/models.py

```diff
@@ -0,0 +1,63 @@
+"""
+Common page model for navigation bars.
+"""
+
+
+# import pytest
+
+# import urllib.parse
+
+from webstr.core import WebstrModel, By, PageElement
+from webstr.common.form import models as form
+
+
+# part of url which is placed after 'usm_web_url'
+# for getting on the described page in the module
+LOCATION = '/#'
```

Comment:
None, but the idea is that you can check the URL of the page if you want, just pul usm_web_url + LOCATION and check it.